main.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup
+
+url = 'https://pogoda.meta.ua/ua/Kyivska/Kyivskiy/Kyiv/'
+
+
+def temperature():
+    sourse = requests.get(url)
+    page = sourse.text
+    feaures = "html.parser"
+    soup = BeautifulSoup(page, feaures)
+    today_temp = soup.find('div', {'class': 'city__main-temp'})
+    today_temp = today_temp.text
+    tt = today_temp[:2]
+    return tt
+
+
+def real_temp():
+    sourse = requests.get(url)
+    page = sourse.text
+    feaures = "html.parser"
+    soup = BeautifulSoup(page, feaures)
```

Comment:
This code duplicates the same piece from the previos function. I suggest moving it to a separate method called `load_weather_data` and reuse in both methods.